mipdb/tables.py

```diff
@@ -334,3 +336,10 @@ def insert_values(self, values, db: Union[DataBase, Connection]):
 
     def get_dataset_enums(self):
         return json.loads(self.table["dataset"].metadata)["enumerations"]
+
+    def get_numpy_type_per_cde(self):
+        mapping = {"text": str, "int": pd.Int64Dtype(), "real": np.float64}
```

Comment:
Are we sure that the `sql_type` can only be text, int, real? What if these are uppercase, do we validate them somewhere?